examples/kitchen_sink.rs

```diff
@@ -0,0 +1,13 @@
+// An example of the avaiable functionality in term
```

Comment:
Not entirely accurate... (to me, this implies that this is *all* available functionality).